external/CMakeLists.txt

```diff
@@ -65,6 +65,12 @@ if (${ENABLE_SPIRV_CODEGEN})
       # This add_compile_options() will only affect the current directory and its subdirectories.
       if (WIN32)
         add_compile_options(/EHs)
+      elseif(CMAKE_COMPILER_IS_GNUCXX)
+        # gcc options to silence warnings not included in the project build files
```

Comment:
IMHO we should use `-w` to suppress all warnings. Selectively disabling them is not quite future proof.

Reply:
That was my initial thought, but since they'd been selectively removed before, I thought I'd be conservative.  If we have no intention of fixing warnings in these external projects, then I agree, it makes sense to just dismiss them all.